src/SOS/SOS.Hosting/CorDebugDataTargetWrapper.cs

```diff
@@ -107,7 +107,7 @@ private int GetPlatform(
                     case Architecture.Arm64:
                         platform = CorDebugPlatform.CORDB_PLATFORM_POSIX_ARM64;
                         break;
-                    case (Architecture)6 /* Architecture.LoongArch64 */:
+                    case Architecture.LoongArch64:
```

Comment:
This isn't going to work since we build and have to build with the .NET 6.0 SDK.